GVFS/GVFS.FunctionalTests/Tests/EnlistmentPerTestCase/DiskLayoutUpgradeTests.cs

```diff
@@ -0,0 +1,133 @@
+﻿using GVFS.FunctionalTests.FileSystemRunners;
+using GVFS.FunctionalTests.Should;
+using GVFS.FunctionalTests.Tools;
+using GVFS.Tests.Should;
+using System;
+using System.IO;
+
+namespace GVFS.FunctionalTests.Tests.EnlistmentPerTestCase
+{
+    public abstract class DiskLayoutUpgradeTests : TestsWithEnlistmentPerTestCase
+    {
+        protected const string PlaceholderListDatabaseDelimiter = "\0";
+
+        protected static readonly string PlaceholderListDatabaseContent = $@"A .gitignore{PlaceholderListDatabaseDelimiter}E9630E4CF715315FC90D4AEC98E16A7398F8BF64
+A Readme.md{PlaceholderListDatabaseDelimiter}583F1A56DB7CC884D54534C5D9C56B93A1E00A2B

```

Comment:
Use `GVFSHelpers.PlaceholderFieldDelimiter` and remove `PlaceholderListDatabaseDelimiter`?